index.js

```diff
@@ -236,7 +439,7 @@ Ycb.prototype = {
     _contextToObject: function(context) {
         var contextObj = {};
         for(var i=0; i<context.length; i++) {
-            if(context[i] !== '0') {
+            if(context[i] !== 0) {
```

Comment:
why is it a number now?

Reply:
The context array is built from the map keys which are numbers, before these were js objects so they were getting converted to strings.